src/Lime.Protocol.UnitTests/DocumentExtensionsTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using Lime.Messaging.Contents;
+using NUnit.Framework;
+using Shouldly;
+
+namespace Lime.Protocol.UnitTests
+{
+    [TestFixture]
+    public class DocumentExtensionsTests
+    {
+        [Test]
+        [Category("ToDocumentContainer")]
```

Comment:
Since all test methods have the same Category attribute, remove from them an put it on the class/testfixture